data/items/items.xml

```diff
@@ -22329,7 +22329,7 @@
 			<attribute key="unproperly" value="true"/>
 			<attribute key="weaponType" value="axe"/>
 			<attribute key="slot" value="hand"/>
-			<attribute key="chain" value="0.9"/>
+			<attribute key="chain" value="1"/>
```

Comment:
Not sure why it would be in the main branch if it is a demonstration. But if it is testing doubles then perhaps it should be 1.0 as the pugixml library as_bool function returns false if 0 is found. 

![image](https://github.com/opentibiabr/canary/assets/4534564/69e1691b-e701-40cb-8b7a-f79515fec744)

So when this code is hit it writes out the warning and disables the chain attribute on the axe.

![image](https://github.com/opentibiabr/canary/assets/4534564/e392ee14-2366-4cc1-b512-0f11bf0546c6)

Reply:
Please, revert this, I'm already fixed here: https://github.com/opentibiabr/canary/pull/2421

Reply:
Reverted hellforged axe chain value back to 0.9